Lib/pdb.py

```diff
@@ -339,7 +340,7 @@ class Pdb(bdb.Bdb, cmd.Cmd):
     _last_pdb_instance = None
 
     def __init__(self, completekey='tab', stdin=None, stdout=None, skip=None,
-                 nosigint=False, readrc=True, mode=None, backend=None):
+                 nosigint=False, readrc=True, mode=None, backend=None, colorize=False):
```

Comment:
How about making the argument keyword-only since it's a boolean?
```suggestion
                 nosigint=False, readrc=True, mode=None, backend=None, *, colorize=False):
```

Reply:
I believe most of the arguments here should be keyword only. However, I don't think is particularly relevant in this PR. Adding the `*` in this random places is a bit weird because if we want to add it, it should be earlier.

Reply:
Fair enough, it's not a big deal anyway :slightly_smiling_face: